src/client.ts

```diff
@@ -38,21 +39,10 @@ export interface PredictOptions {
    * This performs a cropping operation on the server and will increase response time.
    */
   cropper?: boolean;
+  /** Page-specific options. */
```

Comment:
```suggestion
  /** If set, remove pages from the document as specified. This is done before sending the file to the server and is useful to avoid page limitations. */
```